app/cdap/components/Breadcrumb/index.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Typography } from '@material-ui/core';
+import Breadcrumbs from '@material-ui/core/Breadcrumbs';
+import { grey } from '@material-ui/core/colors';
+import NavigateNextIcon from '@material-ui/icons/NavigateNext';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+const PREFIX = 'features.WranglerNewUI.Breadcrumb';
+
+export interface IBreadcrumbItemProps {
+  link?: string;
+  label: string;
+}
+
+export interface IBreadcrumbProps {
+  breadcrumbsList: IBreadcrumbItemProps[];
+}
+
+const CustomizedBreadcrumb = styled(Breadcrumbs)`
```

Comment:
nit: we normally use `Styled<Componet Name>` instead of `Customized Component Name>`. Applies to the component names below as well. Won't repeat the comment.

Reply:
changed the customized word to styled for the styled-components